DGL_testing/environment.yaml

```diff
@@ -0,0 +1,17 @@
+name: dgl_ptm_gpu
```

Comment:
I noticed you use this environment file here to install cuda as an optional dependency in case users want to use GPU. I found it confusing to have two environment files for one package. The better approach is to use `pyproject.toml` file for this purpose. I understand that it is out of the scope of this PR. and made issue #57 for this. For now, can you add a comment at the top of this file and explain the purpose of this environment?

Reply:
I have made an attempt at merging the env file for now to avoid confusion. There is a commented package for CPU and GPU depending on choice of device. ALthough I do think there is a better solution here. Have to test it more on snellius to find the optimal install.

Reply:
I have tested the environment file on snellius and with the GPU installation (assuming you have NVIDIA GPUs available) both cpu and gpu runs are succesful without having to install additional packages. 

If NVIDIA drivers are not available then the cpu only dgl needs to be installed.